production/technical_userguide.md

```diff
@@ -67,15 +65,42 @@ Client Secret:
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-In the following cURL command, notice that we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which performs Basic authentication by passing in an additional header**
+
+Format:
 
 ```
-curl -d '' -X POST "https://api.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic MDk4NjlhN2YtNDZjZS00OTA4LWE5MTQtNjEyOWQwODBhMmFlOjY0OTE2ZmU5NmY3MWFkYzc5YzU3MzVlNDlmNGU3MmYxOGZmOTQxZDBkZDYyY2Y0M2VlMWFlMDg1N2UyMDRmMTczYmExMGU0MjUwYzEyYzQ4"
+curl -H "authorization: Basic [base64-encoded clientId:secret]"
 ```
 
-**Response**
+Example:
+
+```
+curl -d '' -X POST "https://sandbox.bcda.cms.gov/auth/token" -H "accept: application/json" -H "authorization: Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOm\
+Y5NzgwZDMyMzU4OGYxY2RmYzNlNjNlOTVhOGNiZGNkZDQ3NjAy\
+ZmY0OGE1MzdiNTFkYzVkNzgzNGJmNDY2NDE2YTcxNmJkNDUwOGU5MDRh"
+```
+
+**Option 2, which does only requires you to submit your clientId and secret**
+
 
-You will receive a `200 OK` response and an access token if your credentials were accepted, with a body like so:
+Format:
+
+```
+curl --user clientId:secret
+```
+
+Example:
+
+```
+curl --user 3841c594-a8c0-41e5-98cc-38bb45360d3c:f9780d323588f1cdfc3e63e95a8cbdcdd47602ff48a537b51dc5d7834bf466416a716bd4508e904a
```

Comment:
The example should probably include the appropriate headers, body, action, and destination address; similar to the example in option 1.

Reply:
Keeping open; related to conversation in https://github.com/CMSgov/bcda-static-site/pull/57#discussion_r415877240

Reply:
Keeping open, as it relates to the conversation in https://github.com/CMSgov/bcda-static-site/pull/57#discussion_r415877240

Reply:
Still need to make this a complete `curl` command, so that it will actually send data to our `/auth/token` endpoint.

Reply:
Accepted, resolved. I tried the cURL command in terminal and did not get an error message.